Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -2,13 +2,17 @@ Trace-VstsEnteringInvocation $MyInvocation
 Import-VstsLocStrings "$PSScriptRoot\Task.json"
 
 # Get inputs.
-$scriptPath = Get-VstsInput -Name ScriptPath -Require
+$scriptType = Get-VstsInput -Name ScriptType -Require
```

Comment:
when script type is filepath, then shouldn't ScriptPath be required? i.e. `Get-VstsInput -Name ScriptPath -Require`

similar thing for inline scenario